.circleci/config.yml

```diff
@@ -19,7 +19,6 @@ commands:
             rm -f /usr/bin/python3;
             ln -s /usr/bin/python3.7 /usr/bin/python3;
             python3 -m pip -q install matplotlib;
```

Comment:
Do we still use this config for anything? Can we get rid of the entire .circleci folder?

Reply:
still used in nightly regression